docker-compose.yml

```diff
@@ -5,7 +5,11 @@ services:
     build: .
     container_name: knowledge-panel
     volumes:
-      - .:/code
+      - knowledge-panel:/code
```

Comment:
No, this is a not data :-)
We are manually mounting . as /code to avoid rebuilding the docker container on code change while developing.
You should instead move this bind mount definition in docker/dev.yml (this is a dev setting)

Reply:
I understand